parser_test.go

```diff
@@ -181,7 +118,107 @@ func TestExample(t *testing.T) {
                         "description": "Invalid resource identifier"
                     }
                 },
-                "summary": "Get inner foos"
+                "summary": "Get inner foos",
+                "description": " Get Inner Foos"
+            }
+        }
+    },
+    "components": {
+        "schemas": {
+            "DoubleAlias": {
+                "additionalProperties": {
+                    "type": "object"
+                }
+			},
```

Comment:
formatting is slightly off, but @kbrackbill does this look right?

Reply:
looks like this one line is using tabs instead of spaces, you can probably just fix it and it'll be fine